types/d3-color/index.d.ts

```diff
@@ -108,6 +132,12 @@ export interface HCLColorFactory extends Function {
     readonly prototype: HCLColor;
 }
 
+export interface LCHColorFactory extends Function {
```

Comment:
There is no need to extend `Function`. If an interface has call signatures it automatically gets `Function` as an apparent type.